### PR TITLE
[stable] Add a Helm chart for k8s-spot-rescheduler

### PR DESCRIPTION
**Pull request by komljen**
https://github.com/pusher/k8s-spot-rescheduler

### CODE REVIEW

**Comment by unguiculus** [comment]
Please follow our RBAC best practices.

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

File: incubator/k8s-spot-rescheduler/templates/clusterrolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    app: {{ template "k8s-spot-rescheduler.name" . }}
+    chart: {{ template "k8s-spot-rescheduler.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "k8s-spot-rescheduler.fullname" . }}
+subjects:
+- kind: ServiceAccount
```
Comment (unguiculus) [inline incubator/k8s-spot-rescheduler/templates/clusterrolebinding.yaml]: Please choose a consistent list indentation style. It's mostly two spaces, so please apply this everywhere. Don't mix styles.

File: incubator/k8s-spot-rescheduler/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+# Notes TBC
```
Comment (unguiculus) [inline incubator/k8s-spot-rescheduler/templates/NOTES.txt]: Please add some notes.

**Review by unguiculus — CHANGES_REQUESTED** [review]
I'd suggest you move the chart to stable directly.

File: incubator/k8s-spot-rescheduler/templates/clusterrolebinding.yaml (hunk 1 above)
Comment (komljen) [reply incubator/k8s-spot-rescheduler/templates/clusterrolebinding.yaml]: Hm, not sure where I mixed styles? I used 2 spaces everywhere.

Comment (komljen) [reply incubator/k8s-spot-rescheduler/templates/clusterrolebinding.yaml]: Ahh, I see. Will fix.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**